airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle

```diff
@@ -16,9 +16,11 @@ plugins {
     id "java-library"
     id "java-test-fixtures"  // https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures
 }
-// TODO: Remove this after debugging
+
 test {
+    systemProperty 'user.timezone', 'UTC'
     testLogging {
+        // TODO: Remove this after debugging
         showStandardStreams = true
```

Comment:
Should you be commenting this line out? I don't have an opinion either way, I don't know what it does, it just seems like an oversight.

Reply:
Oh, my bad, I didn't realize it had been uncommented a while ago. Please disregard my comment :)